src/components/FeatureViews/AdminPortal/ActionLog.js

```diff
@@ -0,0 +1,482 @@
+                                        value={startDate ? moment(startDate).format("MM/DD/YYYY") : " "}
+                                        onClick={() => setOpenCalendar(true)}
+                                        inputProps={{
+                                            style: {
+                                                padding: 5
+                                            }
+                                        }}
+                                        style={{ paddingLeft: 5, paddingRight: 5, width: 90 }} />
+                                </Grid>
+                                <Grid>
+                                    To <TextField
+                                        value={endDate ? moment(endDate).format("MM/DD/YYYY") : " "}
+                                        variant="outlined"
+                                        onClick={() => setOpenCalendar(true)}
+                                        inputProps={{
+                                            style: {
+                                                padding: 5
+                                            }
+                                        }}
+                                        style={{ paddingLeft: 5, width: 90 }} />
+                                </Grid>
+                            </Grid>
+                        </TableCell>
+                        <TableCell style={{ width: 130 }}>
+                            <Grid container>
+                                <Grid>
+                                    User
+                                </Grid>
+                                <Grid>
+                                    <SwapVertIcon onClick={handleUserSort} style={{ color: "grey" }} />
+                                </Grid>
+                            </Grid>
+                            <Grid>
+                                <FormControl
+                                    variant="outlined"
+                                >
+                                    <InputLabel shrink={false} style={{ lineHeight: "5px" }} margin="dense" >{userType ? "" : "User Type"}</InputLabel>
+                                    <Select
+                                        id="select"
+                                        labelId="label"
+                                        value={userType}
+                                        onChange={handleUserSelection}
+                                        style={{ width: 130 }}
+                                        classes={{
+                                            outlined: classes.outlined,
+                                            root: classes.root,
+                                            "MuiInputBase-input": classes.MuiInputBase
+                                        }}
+                                    >
+                                        {userOptions && userOptions.map((userType) =>
```

Comment:
refer to my comment earlier - we shouldn't be doing this `userOptions && userOptions.map()...` 

use the `loading` variable given by the useQuery hook to validate if `userOptions`/the data for userOptions has loaded